### DISCUSSION TITLE
I am unable to verify my domain using TXT record it's been over 48 hours

### DISCUSSION THREAD

**Discussion by sendevops33** [vercel/community#1243] [Help] [answered]
### Summary

-I had a project which was using my domain   lets call it example.ai  , 
-now I deleted the domain settings from that project and created a new poject
- I want to use this domain for my new project
- But I am unable to verify the domain though TXT record even after 48 hours
- 
![spoofsense domain verification error](https://user-images.githubusercontent.com/119293710/211755915-64685e53-cff6-4347-abb8-5053123b387a.jpeg)

**Comment by swarnava** [comment, marked as answer]
Hi,

You have currently proved ownership for www.spoofsense.com and still yet to setup value for spoofsense.com: https://dnschecker.org/#TXT/_vercel.spoofsense.com.

Please update the TXT record with suggested value.
